go/libraries/doltcore/remotestorage/chunk_store.go

```diff
@@ -249,8 +249,7 @@ func (dcs *DoltChunkStore) GetManyCompressed(ctx context.Context, hashes hash.Ha
 }
 
 const (
-	getLocsBatchSize      = 32 * 1024
-	getLocsMaxConcurrency = 4
+	getLocsBatchSize = (4 * 1024) / 20
```

Comment:
20 because that's the hash size?  Would probably just change this to:

```suggestion
	getLocsBatchSize = 256
```